hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/regionserver/TestCompactionTool.java

```diff
@@ -60,12 +57,6 @@
   public void setUp() throws Exception {
     this.testUtil.startMiniCluster();
     testUtil.createTable(tableName, HBaseTestingUtility.fam1);
-    String defaultFS = testUtil.getMiniHBaseCluster().getConfiguration().get("fs.defaultFS");
-    Configuration config = HBaseConfiguration.create();
-    config.set("fs.defaultFS", defaultFS);
-    String configPath = this.getClass().getClassLoader()
-      .getResource("hbase-site.xml").getFile();
-    config.writeXml(new FileOutputStream(new File(configPath)));
```

Comment:
@wchevreuil can you comment as to why this test needed to write out it's `hbase-site.xml`? Presumably it was done for a reason and it's not enough to simply not write it out.